ebuild-writing/common-mistakes/text.xml

```diff
@@ -259,7 +259,7 @@ part.
 </body>
 </subsection>
 <subsection>
-<title>DEPEND is incomplete</title>
+<title>(B/R)DEPEND is incomplete</title>
```

Comment:
Is this heading change necessary? It will break external links to the section.

Maybe mention instead in the first section that this applies to all dependency variables?

Reply:
Done.